File: src/log.js
export function error(msg) {
    new vim.api.nvim_err_writeln();
}

export function log(msg) {
    if (vim.g.context_menu_config.debug) {
        // TODO: implement
        // vim.notify(msg);
    }
}

export function trace(what) {
    require("notify")(new vim.inspect(what))
}

File: lua/context-menu/log.lua
local _M = {}

local function error(msg)
	vim.api.nvim_err_writeln()
end;
local function log(msg)
	if vim.g.context_menu_config.debug then
	end
end;
local function trace(what)
	require("notify")(vim.inspect(what))
end
_M.error = error;
_M.log = log;
_M.trace = trace
return _M
